src/vs/workbench/contrib/debug/browser/media/repl.css

```diff
@@ -102,7 +102,46 @@
+/* because they can exist at same time we need all the possible underline(or double-underline),overline and strike-through combinations */
+.monaco-workbench .repl .repl-tree .output.expression .code-overline.code-underline.code-strike-through { text-decoration: overline underline line-through; text-decoration-style:solid; }
+.monaco-workbench .repl .repl-tree .output.expression .code-overline.code-underline { text-decoration: overline underline; text-decoration-style:solid; }
+.monaco-workbench .repl .repl-tree .output.expression .code-overline.code-strike-through { text-decoration: overline line-through; text-decoration-style:solid; }
+.monaco-workbench .repl .repl-tree .output.expression .code-underline.code-strike-through { text-decoration: underline line-through; text-decoration-style:solid; }
+.monaco-workbench .repl .repl-tree .output.expression .code-overline.code-double-underline.code-strike-through { text-decoration: overline underline line-through; text-decoration-style:double; }
+.monaco-workbench .repl .repl-tree .output.expression .code-overline.code-double-underline { text-decoration: overline underline; text-decoration-style:double; }
+.monaco-workbench .repl .repl-tree .output.expression .code-double-underline.code-strike-through { text-decoration: underline line-through; text-decoration-style:double; }
+.monaco-workbench .repl .repl-tree .output.expression .code-dim	{ opacity: 0.4; }
+.monaco-workbench .repl .repl-tree .output.expression .code-hidden { opacity: 0; }
+.monaco-workbench .repl .repl-tree .output.expression .code-blink { animation: code-blink-key 1s cubic-bezier(1, 0, 0, 1) infinite alternate; }
+.monaco-workbench .repl .repl-tree .output.expression .code-rapid-blink { animation: code-blink-key 0.3s cubic-bezier(1, 0, 0, 1) infinite alternate; }
+@keyframes code-blink-key {
+	to { opacity: 0.4; }
+}
+.monaco-workbench .repl .repl-tree .output.expression .code-subscript { vertical-align: sub; font-size: smaller; line-height: normal; }
+.monaco-workbench .repl .repl-tree .output.expression .code-superscript { vertical-align: super; font-size: smaller; line-height: normal; }
+/**
+ * Alternate code-font-# classes, note the font-family stacks here are somewhat arbitrary but will resolve to different 'standard' css fonts
+ * or are font-family stacks found other places in within VSCode.
+ * code-font-10 is called the 'blackletter' font within ANSI SGR docs so attempt is made to resolve a blackletter font on users system.
+ * ('F25 BlackletterTypewriter' is monospaced Blackletter font used or recommended by other terminal emulators (ie. mintty, etc.)
+ * None of these fonts are required and all font-family stacks will resolve to some font.
+ * User can change the fonts used by these classes custom css using a extension such as Customize UI/Monkey Patch
+ */
+.monaco-workbench .repl .repl-tree .output.expression .code-font-1 { font-family: Verdana,Arial,sans-serif; }
```

Comment:
I am not a fan of hard coding all the fonts inside this CSS.
I suggest to remove the font support for now, and if there is user interest in this we can add it in the future.
Also using an extension to customize the CSS is not recommended by VS Code.

Reply:
I will just make one argument for keeping fonts.  This is for the DEBUG console.   The only purpose of those fonts is to give the developer an option to print out messages in *DIFFERENT* fonts.   For me I felt that having the option to print messages in different fonts was very useful.  I do multi-threaded development and I have found it very useful to have loggers from different threads (as well as async futures) log messages in different fonts to make it easier to go through logs and see what was happening.
    In this context I found that it did not matter *what* the fonts were, it only mattered that I had some different options to use.   I understand that I can use colors or italics (and I do), but I find overuse of colors can be distracting, especially with several thread's log messages interleaved and making a 'rainbow' of colors in the logs.

    ...And that's my argument... lol.  
   I knew there would be push back on the fonts, and I fully expected there was a large chance that I would have to remove the font classes.   No one (typically) likes the idea of a bunch of hardcoded font stacks, but I thought in the context of this being our developer debug console, not for end user use, that there might be a chance that other developers could appreciate the usefulness of being able to use several fonts in your log files.

    If that does not persuade you in any way then I will remove the classes for the font stacks.  Let me know.

Thanks for letting me argue my reasoning for having them there in the first place!

Reply:
I see the use case. However I am not sure how popular it would be and I am not a fan of the approach.
So I suggest the following, just create a new PR after we merge this one with the font changes. If I see big user interest in it in the next year I will merge that one as well.